odoo_initializer/activator.py

```diff
@@ -62,6 +67,17 @@ def start_init(cr):
         BomLoader,
         CashRoundingLoader,
         LanguageLoader,
+        OAuthLoader,
+        StockPickingTypeLoader,
+        StockLocationRouteLoader,
+        IrActionsActWindowLoader,
+        IrModelFieldsLoader,
+        IrModelFieldsSelectionLoader,
+        IrModelLoader,
+        IrUiViewLoader,
```

Comment:
I would put anything that touches on the Model/Fields additions at the top of the list

Reply:
Yeah I tried that but it breaks the initialisation.